server_admin/topics/authentication/flows.adoc

```diff
@@ -132,6 +132,17 @@ Executions have a *Config* menu option to configure the execution. It is also po
 
 Since the order of execution is important, you can move executions and sub-flows up and down within their flows using the up and down buttons beside their names.
 
+[WARNING]
+====
+Make sure to properly test your configuration when you configure the authentication flow to doublecheck there are no security holes in your setup. It is recommended to test various
+corner cases. Like for example the behaviour of authentication for the particular user when you remove various credentials from his account before the authentication.
+
+As an example, when 2nd-factor authenticators, like OTP Form or WebAuthn Authenticator, are configured in the flow as REQUIRED and user does not have credential of particular
```

Comment:
generally replace "like" with "such as" or "for example"
add "the" before "user"